cctbx/euclidean_model_matching.py

```diff
@@ -249,8 +251,13 @@ def generate_singles(n, i):
   return singles
 
 def pair_sort_function(pair_a, pair_b):
-  from past.builtins import cmp
-  return cmp(pair_a[0], pair_b[0])
+  # Deprecated. Do not use
+  if pair_a[0] < pair_b[0]:
+    return -1
+  elif pair_a[0] > pair_b[0]:
+    return 1
+  else:
+    return 0
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.